go/vt/sqlparser/comments.go

```diff
@@ -157,3 +158,90 @@ func ExtractMysqlComment(sql string) (version string, innerSQL string) {
 
 	return version, innerSQL
 }
+
+const commentDirectivePreamble = "/*vt!"
```

Comment:
According to this page https://dev.mysql.com/doc/refman/5.7/en/comments.html, `/*!` is for injecting sql tokens that will be visible only to mysql. For hints, they use `/*+`. In that spirit, this should instead be `/*vt+`.